1136/en.md

```diff
@@ -0,0 +1,138 @@
+# 1136 - Division by 3
```

Comment:
Please keep the format for the heading like this - https://github.com/lightoj-dev/problem-tutorials/pull/32/files#diff-94aeb337c51c48839ef777311d73c6d0fb848fcce5914df92d7f1e2973136b9cR1